### DISCUSSION TITLE
Hide top and bottom bars in landscape view

### DISCUSSION THREAD

**Discussion by nisedo** [community/community#136920] [Mobile]
Hi, it would be great to be able to hide the top and bottom bars when reading code in landscape mode because they take up a lot of space on mobile.

![image](https://github.com/user-attachments/assets/2f47eb88-c0e8-4101-b600-96f8dc2c8fb0)

<details>
GitHub for iOS, Version 1.174.0 (139241358)
Device: iPhone 15 Pro (iOS 17.5.1)
</details>

**Comment by eliperkins** [comment]
Hey @nisedo, this is great feedback. I can see hiding the top and bottom nav bars being able to give more room to the code itself! I've created an issue for this in our team's tracking repo. I'll work with the team to figure out how this feature can be implemented, and keep you posted.

Thanks for helping improve GitHub Mobile!

**Reply by nisedo** [reply]
Hi, any update on this?
